app/views/admin/pages/_form.html.erb

```diff
@@ -48,15 +48,15 @@
           <span class="badge badge-<%= FeatureFlag.exist?(@page.feature_flag_name) ? "success" : "warning" %>">
             <%= FeatureFlag.exist?(@page.feature_flag_name) ? "Present" : "Not Present" %>
           </span>
-          </br>
+          <br>
```

Comment:
Good catch. You can add them to the end though.

```suggestion
          <br />
```

Reply:
This is true, but I don't think it's necessary to do since they're self-closing! Correct me if I'm wrong, though. 😄

Reply:
It's definitely not a hard requirement as `<br>`, `<br />` are both valid. I'm used to the self-closing because in Preact land, it causes JSX syntax errors.

Reply:
Oh, good to know @nickytonline!